.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -205,8 +338,57 @@ dynamodb_PutItem:
             - description:
               snippet_tags:
                 - dynamodb.rust.add-item
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-table-read-write.html#dynamodb-example-table-read-write-writing-an-item
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createclientv3
+            - description: Add the item.
+              snippet_tags:
+                - dynamodb.JavaScript.item.putItemV3
+        - sdk_version: 2
+          github: javascript/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v2/developer-guide/dynamodb-example-table-read-write.html#dynamodb-example-table-read-write-writing-an-item
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.JavaScript.item.putItem
   services:
     dynamodb: {PutItem}
+dynamodb_Put:
+  title: Put an item in a &DDB; table using the &DDB; document client using an &AWS; SDK
+  title_abbrev: Put an item in a table using the document client
+  synopsis: put an item in a &DDB; table using the &DDB; document client.
+  category:
+  languages:
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide: sdk-for-javascript/v3/developer-guide/dynamodb-example-dynamodb-utilities.html#dynamodb-example-document-client-put
+          excerpts:
+            - description: Create the &DDB; document client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createdocclientv3
+            - description: Create the &DDB; client.
+              snippet_tags:
+                - dynamodb.JavaScript.tables.createdocclientv3

```

Comment:
Wrong snippet: it's for creating the document client and you meant the standard client.
Also, I'd move the standard client block above the document client block because it reads better.

Reply:
fixed